civis/futures.py

```diff
@@ -1,33 +1,36 @@
 from __future__ import absolute_import
+from builtins import super
 
 from abc import ABCMeta, abstractmethod
-from builtins import super
 from concurrent.futures import Executor
 from concurrent import futures
 import copy
 import datetime
 import logging
 import time
 import threading
+import warnings
 
 import six
 
 from civis import APIClient
-from civis.base import DONE
-from civis.polling import PollableResult, _ResultPollingThread
+from civis.base import CivisJobFailure, FINISHED, STATE_TRANS, FAILED, DONE
```

Comment:
I think having them in base is better. These are core conventions to our API. Either way, we need to preserve their presence in base to avoid breaking stuff. Thus moving them here is fine as long as we import them back.

Reply:
Okay. If you'd rather leave them in `base`, that works for me.